presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkMemoryBasedBroadcastDependency.java

```diff
@@ -81,6 +85,32 @@ public Broadcast<List<PrestoSparkSerializedPage>> executeBroadcast(JavaSparkCont
         return broadcastVariable;
     }
 
+    private long getNativeMaxSerializedBytes(List<PrestoSparkSerializedPage> prestoSparkSerializedPages, PagesSerde pagesSerde)
+    {
+        return prestoSparkSerializedPages.stream()
+                .map(prestoSparkSerializedPage -> PrestoSparkUtils.toSerializedPage(prestoSparkSerializedPage))
+                .map(serializedPage -> pagesSerde.deserialize(serializedPage))
+                // Extract sum of maxSerializedSize.
+                .flatMap(page -> IntStream.range(0, page.getPositionCount())
+                        .mapToObj(position -> BigintType.BIGINT.getObjectValue(null, page.getBlock(1), position)))
+                .mapToLong(Long.class::cast)
+                .sum();
+    }
+
+    private long getUncompressedBroadcastSizeInBytes(List<PrestoSparkSerializedPage> broadcastValue)
```

Comment:
Nit: change parameter name to `prestoSparkSerializedPages` as `broadcastValue` has no meaning inside of this function